.travis.yml

```diff
@@ -1,37 +1,83 @@
-#if: type = pull_request
+if: type = pull_request
```

Comment:
Worth noting that we don't want this always restricted to PRs, as we'll want deployments to run on the master branch